Content.Server/EntityEffects/Effects/AddStealth.cs

```diff
@@ -0,0 +1,31 @@
+using Content.Server.Stealth;
+using Content.Shared.EntityEffects;
+using JetBrains.Annotations;
+using Robust.Shared.Prototypes;
+
+namespace Content.Server.EntityEffects.Effects;
+
+/// <summary>
+/// Adds a temporary invisibility effect
+/// </summary>
+[UsedImplicitly]
+public sealed partial class AddStealth : EntityEffect
+{
+    [DataField]
+    public float Time = 1;
+
+    protected override string? ReagentEffectGuidebookText(IPrototypeManager prototype, IEntitySystemManager entSys)
+        => Loc.GetString("reagent-effect-guidebook-add-stealth", ("chance", Probability));
```

Comment:
you should integrate the time value into the guidebook text. You can probably just do something like "makes invisible for X seconds per 1u".